timesketch/api/v1/resources/sketch.py

```diff
@@ -423,6 +423,15 @@ def get(self, sketch_id):
             }
             stories.append(story)
 
+        analyzers = analyzer_manager.AnalysisManager.get_analyzers()
+        analyzers_detail = []
+        for analyzer_name, analyzer_class in analyzers:
+            analyzers_detail.append({
+                'name': analyzer_name,
+                'display_name': analyzer_class.DISPLAY_NAME,
+                'description': analyzer_class.DESCRIPTION
+            })
+
```

Comment:
this is also another API call, '{api_root}/sketches/{sketch.id}/analyzer/` 

it returns ever so slightly different output, but can we not just use that? instead of adding this to all sketch data? which we were trying to reduce the other day

Reply:
Good catch!

In an early iteration of the PR I still used the dropdown component and needed this but I have since removed that. I have refactored this to use the API call you mentioned, extended it with more information. This affects the API client, but on the end user part when listing the analyzers. I have not found anything in picatrix so we should be OK.